SPEC.md

```diff
@@ -146,18 +148,68 @@ Plugins that consume any of these configuration keys should respect their intend
 Plugins may define additional fields that they accept and may generate an error if called with unknown fields. Runtimes must preserve unknown fields in plugin configuration objects when transforming for execution.
 
 #### Example configuration
+Network configuration with no inlined plugin confs, and two loaded plugin confs:
+`/etc/cni/net.d/10-dbnet.conf`:
 ```jsonc
 {
   "cniVersion": "1.1.0",
   "cniVersions": ["0.3.1", "0.4.0", "1.0.0", "1.1.0"],
   "name": "dbnet",
-  "plugins": [
+  "loadOnlyInlinedPlugins": false,
+}
+```
+
+`/etc/cni/net.d/dbnet/5-bridge.conf`:
+```jsonc
+{
+  "type": "bridge",
+  // plugin specific parameters
+  "bridge": "cni0",
+  "keyA": ["some more", "plugin specific", "configuration"],
+
+  "ipam": {
+    "type": "host-local",
+    // ipam specific
+    "subnet": "10.1.0.0/16",
+    "gateway": "10.1.0.1",
+    "routes": [
+        {"dst": "0.0.0.0/0"}
+    ]
+  },
+  "dns": {
+    "nameservers": [ "10.1.0.1" ]
+  }
+}
+```
+
+`/etc/cni/net.d/dbnet/10-tuning.conf`:
+```jsonc
+{
+  "type": "tuning",
+  "capabilities": {
+    "mac": true
+  },
+  "sysctl": {
+    "net.core.somaxconn": "500"
+  }
+}
+```
+
+Network configuration with one inlined plugin conf, and one loaded plugin conf:
```

Comment:
Do you want to provide a bit more context for this example? Or maybe just drop it?

Reply:
I am leaning towards dropping it and moving it to https://github.com/containernetworking/cni/pull/1081

Reply:
Heck yeah, totally approve.

Reply:
...but given that we reference it in several existing spots, I'd rather do that in #1081